modules/ROOT/pages/cassandra/cassandra-xml-maven.adoc

```diff
@@ -0,0 +1,59 @@
+= XML and Maven Support - Mule 4
```

Comment:
Again, if we mention Mule 4, for SEO we should also have Cassandra in the title.